tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -852,7 +859,40 @@ const reducer = createReducer(
         }
       }
 
-      const nextState = {
+      if (response.runToSeries) {
+        for (let runId in response.runToSeries) {
+          const cardMetadata: CardMetadata = {
+            plugin,
+            tag,
+            runId,
+          };
+          if (sample) {
+            cardMetadata.sample = sample;
+          }
+
+          const cardId = getCardId(cardMetadata);
+          const cardTimeSeriesLoadable = getTimeSeriesLoadable(
+            nextTimeSeriesData,
+            cardMetadata.plugin,
+            cardMetadata.tag,
+            cardMetadata.sample
+          );
+          if (cardTimeSeriesLoadable) {
+            const nextMinMax = generateCardMinMaxStep(
+              cardTimeSeriesLoadable.runToSeries
+            );
+            nextCardToMinMax.set(cardId, nextMinMax);
+            nextCardToTimeSeletion.set(cardId, {
```

Comment:
I'm not certain but I don't think this is right.  What if user presses the reload button in the header (or even has auto-reload turned on) and kicks off another series of time series request/responses? Then will their previously-selected time selection be reset when the next response comes in?